static/js/settings.js

```diff
@@ -114,6 +119,71 @@ var SettingsScreen = {
     });
   },
 
+  showAdapterSettings: function() {
+    this.adapterSettings.classList.remove('hidden');
+    const opts = {
+      headers: {
+        'Accept': 'application/json'
+      }
+    };
+    fetch('/settings/adapters', opts).then(function (response) {
+      return response.json();
+    }).then(function (body) {
+      if (!body) {
+        return;
+      }
+
+      const ignored = [
+        'adapters.org.mozilla-iot.adapters.example-plugin',
+        'adapters.org.mozilla-iot.adapters.mock',
```

Comment:
I'd rather not see this hard coded. Why shouldn't the example plugin show up in the list (if it present)?